README.md

```diff
@@ -51,319 +34,113 @@ That kind of tooling is not available to shipping applications today, but that d
 mean we can't take a small step in that general direction while the Rust and Wasm ecosystem
 continues to evolve.
 
-### Key Ideas
-
-* Specify the component API using an abstract *Interface Definition Language*.
-* When implementing the component:
-  * Process the IDL into some Rust code scaffolding to define the FFI, data classes, etc.
-  * Have the component crate `!include()` the scaffolding and fill in the implementation.
-* When using the component:
-  * Process the IDL to produce FFI bindings in your language of choice
-  * Use some runtime helpers to hook it up to the compiled FFI from the component crate.
-
-
-## Status
-
-This is all very experimental and incomplete, but we do have some basic examples working, implementing
-functions in Rust and calling them from Kotlin, Swift, and Python.
-Take a look in the [`./examples/`](./examples/) directory to see them in action.
-
-## Component Interface Definition
```

Comment:
I've removed a *lot* of details here. Most of them were either already documented elsewhere (e.g. in the comments on individual items in the `ComponentInterface` code) or seemed to have out-lived their relevance (there are many ways we're diverging from WASM Interface Types, for example).